### PR TITLE
Restore updates

### PR DESCRIPTION
**Pull request by andrewelamb**

### CODE REVIEW

File: R/app_ui.R [hunk 1]
```diff
@@ -24,6 +24,11 @@ app_ui <- function() {
         projectlive.modules::summary_snapshot_module_ui("summary_snapshot_ui_1"),
         icon = shiny::icon("chart-area")
       ),
+      shiny::tabPanel(
+        "Studies",
+        projectlive.modules::study_summary_module_ui("study_summary_ui_1"),
```
Comment (jaybee84) [inline R/app_ui.R:29]: You may need to modify the y-axis limits on the publication plot to be accomodative of >5 publications :)

**Review by jaybee84 — CHANGES_REQUESTED** [review]
Looks great! just a couple of suggestions :

1. The resources generated plot in summary_snapshot is not filling up the box ... I took a look at the [plot function](https://github.com/Sage-Bionetworks/projectlive.modules/blob/main/R/plot_functions.R), and it seems you might need to add in `plotly::layout(autosize =T)` to the create_resources_generated_plot
2. Lets remove the `unannotated` theme from the resources generated plot

File: R/app_ui.R (hunk 1 above)
Comment (andrewelamb) [reply R/app_ui.R:29]: @jaybee84 https://github.com/Sage-Bionetworks/projectlive.modules/blob/7a4a6325f547f255ec15f34f4c8d1d4eac74ef9c/R/plot_functions.R#L43

Comment (andrewelamb) [reply R/app_ui.R:29]: @jaybee84 All plots using create_plot_with_config() have plotly::layout(autosize =T) added to them.

Comment (andrewelamb) [reply R/app_ui.R:29]: I've fixed y-axis issues. The plot function had y-axis limit of 10 in one spot, and 5 in another. I'm not sure how those got there, maybe they were in the original code I used to create them, and just got left in?

Comment (jaybee84) [reply R/app_ui.R:29]: hmm.. it weird that even with layout:: autosize , the resources generated plot is skewed to one side of the tabpanel box... I know we encounted this earlier too but AFAIK the layout had resolved it. 

Any idea why this is happening for that one plot? The same function seems to work well for projectLive-NF though..

Comment (jaybee84) [reply R/app_ui.R:29]: Also, can you remove the unannotated (highest bar) from the `resources generated` plot? That would help us visualize the other bars better

Comment (andrewelamb) [reply R/app_ui.R:29]: Not sure about the plots. I've given up trying to get ggplotly plots to layout correctly. :)

Comment (jaybee84) [reply R/app_ui.R:29]: The publication status plot in the studies module still has the yaxis limits 0-5

Comment (jaybee84) [reply R/app_ui.R:29]: oh wait I might need to update my projectlive.modules

Comment (andrewelamb) [reply R/app_ui.R:29]: OK I've made some changes, please try them.

Comment (jaybee84) [reply R/app_ui.R:29]: Looks great!

Did you do something to modify the `resources_generated` plot? It looks like it is working well now.

**Review by jaybee84 — APPROVED** [review]

**Pull request merged**